CHANGELOG.md

```diff
@@ -2,7 +2,20 @@
 
 Full documentation for hipTensor is available at [rocm.docs.amd.com/projects/hiptensor](https://rocm.docs.amd.com/projects/hipTensor/en/latest/index.html).
 
-## (Unreleased) hipTensor 1.1.0 for ROCm 6.0.0
+## (Unreleased) hipTensor 1.2.0 for ROCm 6.1.0
+
+### Additions
+
+* API support
+  * Permutation of rank 4 tensors
+* Data type support
+  * Contraction - new supported data type f16, bf16, complex f32, complex f64
+
```

Comment:
Added Scale and Bilinear Contraction samples for complex float types.

Reply:
Added Permutation tests for F16, F32 types

Reply:
done